gui/wxpython/startup/guiutils.py

```diff
@@ -279,6 +318,21 @@ def rename_location_interactively(guiparent, grassdb, location):
     return newlocation
 
 
+def download_location_interactively(guiparent, grassdb):
```

Comment:
It would make sense for this function to have the same interface, esp., the return value as `create_location_interactively()`, so that they can be used interchangeably and that we can change the implementation here and the rest of the code just works. Although the functionality is not here and we won't be adding it right now, creating another mapset here actually makes more sense than when creating an empty location. So, the return value right now should be tuple `(grassdb, location, "PERMANENT")`.

You should describe in documentation, when it returns invalid values and what happens on error or cancel. It should fit in terms of interface with the `create_location_interactively()`.

Reply:
if loc_download.GetLocation() is not None:
        result = (grassdb, loc_download.GetLocation(), "PERMANENT")

Documentation added.

Reply:
Same here, documentation of the return value needs to be in the docstring because it documents the interface.

Reply:
Docstring added.